tests/generate-data/package-lock.json

```diff
@@ -12,19 +12,19 @@
       }
     },
     "@babel/core": {
```

Comment:
Is there a reason `tests/generate-data` needs to be its own package? Rather than making whatever it needs a devDependency in the top-level `package.json`?

Reply:
Personally, I don't like to mix generating test data with app source code. If you want you can merge this project into the main one

Reply:
Created a related issue https://github.com/danvk/source-map-explorer/issues/141